stdlib/src/utils/_format.mojo

```diff
@@ -147,3 +147,83 @@ fn write_to[*Ts: Formattable](inout writer: Formatter, *args: *Ts):
         arg.format_to(writer)
 
     args.each[write_arg]()
+
+
+# ===----------------------------------------------------------------------===#
+# Format method structures
+# ===----------------------------------------------------------------------===#
+
+
+@value
```

Comment:
I think this can live in `string.mojo` as `_FormatCurlyEntry`. There is no need to add a new file in `utils` at this point.

Reply:
Yep, this review can be implemented